src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -237,69 +235,69 @@ public void run() {
   }
 
   public void addMapSelectionListener(final MapSelectionListener listener) {
-    m_mapSelectionListeners.add(listener);
+    mapSelectionListeners.add(listener);
   }
 
   public void removeMapSelectionListener(final MapSelectionListener listener) {
-    m_mapSelectionListeners.remove(listener);
+    mapSelectionListeners.remove(listener);
   }
 
   public void addMouseOverUnitListener(final MouseOverUnitListener listener) {
-    m_mouseOverUnitsListeners.add(listener);
+    mouseOverUnitsListeners.add(listener);
   }
 
   public void removeMouseOverUnitListener(final MouseOverUnitListener listener) {
-    m_mouseOverUnitsListeners.remove(listener);
+    mouseOverUnitsListeners.remove(listener);
   }
 
   private void notifyTerritorySelected(final Territory t, final MouseDetails me) {
-    for (final MapSelectionListener msl : m_mapSelectionListeners) {
+    for (final MapSelectionListener msl : mapSelectionListeners) {
       msl.territorySelected(t, me);
     }
   }
 
   private void notifyMouseMoved(final Territory t, final MouseDetails me) {
-    for (final MapSelectionListener msl : m_mapSelectionListeners) {
+    for (final MapSelectionListener msl : mapSelectionListeners) {
```

Comment:
If you can, the variable renaming is nice in PRs on their own.  Allows the review to focus on the functional changes, and does not hold up the renaming in case there is more to do.